### PR TITLE
Updated validation, moved it to shared since used by vpn, other code …

### PR DESCRIPTION
**Pull request by cxhercules**
- Using db rating validation from shared istio project. 
- Updated README to reflect this.
- Renamed so variables.

### CODE REVIEW

File: scripts/create.sh [hunk 1]
```diff
@@ -22,14 +22,14 @@ set -e
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 # Include the user set variables
-# shellcheck source=properties.env
+# shellcheck disable=SC1091,SC1090
```
Comment (chrislovecnm) [inline scripts/create.sh]: Why did you add the disables?  Can we use the `source`?

Comment (cxhercules) [reply scripts/create.sh]: Seemed not to work, but I saw it worked on yours so let me take a look again.
